src/test/regress/sql/single_node.sql

```diff
@@ -121,6 +121,24 @@ ALTER TABLE simple_table_name RENAME CONSTRAINT "looo oooo ooooo ooooooooooooooo
 SET search_path TO single_node;
 DROP SCHEMA  "Quoed.Schema" CASCADE;
 
+-- test citus size functions in transaction with modification
+CREATE TABLE test_citus_size_func (a int);
```

Comment:
Can we also do tests with modification queries instead of DDLs? Such as:
```SQL
BEGIN;
     -- single shard insert
     INSERT INTO test_citus_size_func VALUES ();
   	SELECT citus_table_size('test_citus_size_func');
ROLLBACK;

BEGIN; 

     -- multi shard modification 
     INSERT INTO test_citus_size_func  SELECT * FROM  test_citus_size_func;
   	SELECT citus_table_size('test_citus_size_func');
ROLLBACK;

Reply:
added